pypsrp/client.py

```diff
@@ -286,3 +291,11 @@ def sanitise_clixml(clixml):
             output = Serializer()._deserialize_string("".join(errors))
 
         return output
+
+
+def _expand_remote_path(runspace_pool, path):
+    expand_command = "[System.Environment]::ExpandEnvironmentVariables('%s')" % (path,)
```

Comment:
We need to be careful here if the path contains certain single quote chars as PowerShell itself uses [IsSingleQuote](https://github.com/PowerShell/PowerShell/blob/b7cb335f03fe2992d0cbd61699de9d9aafa1d7c1/src/System.Management.Automation/engine/parser/CharTraits.cs#L265-L272) to start and terminate single quoted strings. Without escaping them I could have path like `C:\Users\Jordan ‘jborean‘ Borean\file.txt` which will break this command because we've now broken the single string.

What we should be doing instead of trying to embed the string to expand in a script is pass it in as an argument/object and use that directly. This way there is no need to escape any characters and the value is used literally. See my below comment for a way I think we should do this.